packages/expo-cli/src/commands/eas-build/build/utils/gradleContent.ts

```diff
@@ -0,0 +1,52 @@
+export default `android {
```

Comment:
This logic looks ok to me. However, I'm leaving the final decision for @wkozyra95.